distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -454,7 +454,7 @@
   # JWT token generator implementation
   generatorImpl="org.wso2.micro.gateway.jwt.generator.MGWJWTGeneratorImpl"
   [jwtGeneratorConfig.claimRetrieval]
-      //todo: add the sample implementation to retrieve claims from API Manager
+      # //todo: add the sample implementation to retrieve claims from API Manager
```

Comment:
This todo is getting shipped with the product. Thats not nice. Shall we remove it
```suggestion
```

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/1346/commits/73a4ab2144823c7c72db1ed2235b1ee3d798eba5